### PR TITLE
Mandrel/20.1 add jdk ea to matrix

### PR DESCRIPTION
**Pull request by zakkak**

### CODE REVIEW

File: .github/workflows/quarkus.yml [hunk 1]
```diff
@@ -37,8 +37,19 @@ env:
...
     runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        jdk: ['OpenJDK-latest', 'OpenJDK-ea']
+        include:
+          - jdk: 'OpenJDK-latest'
+            tag_cmd: "curl https://api.github.com/repos/AdoptOpenJDK/openjdk11-upstream-binaries/releases/latest | jq -r '.tag_name'"
+            tag_suffix: ""
+          - jdk: 'OpenJDK-ea'
```
Comment (jerboaa) [inline .github/workflows/quarkus.yml:50]: I'd go with `OpenJDK-11-latest-ea` and `OpenJDK-11-latest-ga`. They're both latest, just different release types.

**Review by jerboaa — CHANGES_REQUESTED** [review]
Could we simplify this a bit by using the API?

File: .github/workflows/quarkus.yml [hunk 2]
```diff
@@ -37,8 +37,19 @@ env:
...
-    name: Mandrel build
+    name: Mandrel build - ${{ matrix.jdk }}
     runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        jdk: ['OpenJDK-latest', 'OpenJDK-ea']
+        include:
+          - jdk: 'OpenJDK-latest'
+            tag_cmd: "curl https://api.github.com/repos/AdoptOpenJDK/openjdk11-upstream-binaries/releases/latest | jq -r '.tag_name'"
```
[the remark this answers is not in the document]
Comment (zakkak) [reply .github/workflows/quarkus.yml:48]: > Could we simplify this a bit by using the API?

Yes now that you provided examples of how to achieve this :)
I had a look at https://api.adoptopenjdk.net/ and couldn't figure out how to achieve this. (hint: https://api.adoptopenjdk.net/ v3 needs better docs)

[the remark this answers is not in the document]
Comment (zakkak) [reply .github/workflows/quarkus.yml]: > We could use `java -version` somewhere so as to know which version it built with.

We do this already [here](https://github.com/graalvm/mandrel/pull/112/files#diff-dacc38c76a716b588c8615e697332287R83)

File: .github/workflows/quarkus.yml [hunk 3]
```diff
@@ -37,8 +37,17 @@ env:
 
 jobs:
   build-mandrel:
-    name: Mandrel build
+    name: Mandrel build - ${{ matrix.jdk }}
     runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        jdk: ['OpenJDK-latest-GA', 'OpenJDK-latest-EA']
```
Comment (jerboaa) [inline .github/workflows/quarkus.yml:45]: `OpenJDK-11-latest-GA` ? `OpenJDK-latest-GA` would be 14 now and 15 mid Sept.

Comment (zakkak) [reply .github/workflows/quarkus.yml:45]: Good catch

**Review by jerboaa — APPROVED** [review]
LGTM

**Comment by jerboaa** [comment]
Still good.

**Pull request merged**